gammapy/image/radial_profile.py

```diff
@@ -0,0 +1,87 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Image utility functions"""
+import logging
+import numpy as np
+from astropy.coordinates import Angle
+from astropy.wcs.utils import skycoord_to_pixel
+from astropy.table import QTable
+from scipy.ndimage.measurements import labeled_comprehension
+
+__all__ = [
+    'label_image',
+    'radial_profile',
+]
+log = logging.getLogger(__name__)
+
+
+def label_image(image, theta_bin=None, center=None):
+    """
+   Calculate the labeled image to know which pixel is group together for the radial profile.
+
+   Parameters
+   ----------
+   image : `~gammapy.image.SkyMap`
+         Skymap of the image we cant to compute the radial profile
+   theta_bin : `~astropy.coordinates.Angle`
+         Bining for the radial profile
+   center : `~astropy.coordinates.SkyCoord`
+         The `SkyCoord` of the pixel used as the center. The default is
+            None, which then uses the coordinates of the center of the image.
+
+    Returns
+    -------
+    index_map  : `~numpy.arrays`
```

Comment:
Please call the return value `label_image` and make it a SkyImage (copying over the WCS from the input image).